src/Components/Patient/ManagePatients.tsx

```diff
@@ -141,8 +139,6 @@ export const PatientManager = () => {
         updateQuery({ emergency_phone_number: null });
       return;
```

Comment:
To continue with the error, i.e., 8-digit and 9-digit phone numbers are considered valid. This is because it is validating telephone numbers and other numbers that are more than 10 digits 13 digits. Can I get some suggestions to correct this?

Reply:
We don't need to clear the error if we are not creating an error in the first place right?

Reply:
but what about this issue 8 digit and 9 digit numbers are considered as valid because it is validating telephone number also !.

Reply:
You can trigger the search as soon as the number is considered valid. Use the PhoneNumberValidator utility to achieve that.